test/display_test.py

```diff
@@ -334,7 +334,14 @@ def test_gl_get_attribute(self):
         with self.assertRaises(TypeError):
             pygame.display.gl_get_attribute("DUMMY")
 
-    def todo_test_gl_set_attribute(self):
+    @unittest.skipIf(
+        (
+            "skipping for all because some failures on rasppi and maybe other platforms"
+            or os.environ.get("SDL_VIDEODRIVER") == "dummy"
+        ),
+        'OpenGL requires a non-"dummy" SDL_VIDEODRIVER',
+    )
```

Comment:
I don't get it. What's the point of a test that's always skipped?

Reply:
Yeah, it's kind of difficult. Because OpenGL is broken in quite a few places I guess.
But, also maybe if we enable the test then people will fix OpenGL issues more.

Further work would be to slowly find, and special-case-skip all known broken places. But that could take a while getting tests from lots of different system config failures, and be risky/time consuming.

Hrmm. For me, merging the PR as is would be fine.

Reply:
We also have some opengl tag that's used to disable other opengl tests.